src/stampy_nlp/utilities/coda_utils.py

```diff
@@ -44,7 +44,10 @@ def get_df_data():
         values = item['values']
         pageid = str(values['UI ID']).strip()
         status = values['Status']
-        names = [val for val in values.get('All Phrasings', '').split('\n') if val] or [item['name']]
+        names = [item['name']]
+        # add alternate phrasings only if db already populated, else is_similar always false
+        if not delete_all:
+            names = [val for val in values.get('All Phrasings', '').split('\n') if val] or [item['name']]
```

Comment:
Workaround funky initialization state when the db is empty, so ALL alternate phrasings automatically get added at start.